### PR TITLE
Feat: allow for spaces in prompt keys

### PR DESCRIPTION
**Pull request by fjebaker**

### CODE REVIEW

File: server/connections.py
Comment (JR-Mitchell) [inline server/connections.py:1]: I think you need to add new logic for `prompt_length` - at the moment, it includes the space in the length, meaning the players have to type one extra character, which then means their answer will never match (I think)

Comment (JR-Mitchell) [reply server/connections.py:1]: I think `len(self.manager.get_key())` might work

**Review by JR-Mitchell — CHANGES_REQUESTED** [review]
Fix the prompt length issues ^

**Review by RiFactor — APPROVED** [review]

**Review by JR-Mitchell — APPROVED** [review]
yeeeees

**Pull request merged**